src/server/kiwix-serve.cpp

```diff
@@ -422,7 +424,11 @@ int main(int argc, char** argv)
 
     if ( monitorLibrary ) {
       curLibraryFileTimestamp = newestFileTimestamp(libraryPaths);
+#ifndef _WIN32
       libraryMustBeReloaded += curLibraryFileTimestamp > libraryFileTimestamp;
+#else
+      libraryMustBeReloaded |= curLibraryFileTimestamp > libraryFileTimestamp;
+#endif
```

Comment:
Why is this change needed? `+=` should have the needed effect on a `bool` variable too. Or does the compiler issue a warning?

Reply:
> Or does the compiler issue a warning?

It is. And we are compiling with `Werror`.

Reply:
We better address that problem without conditional compilation. For example:

```c++
if ( !libraryMustBeReloaded ) { 
  libraryMustBeReloaded = curLibraryFileTimestamp > libraryFileTimestamp;
}
```